src/pages/MainTest.jsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from "react-router-dom";
+import KDH from "../components/kdh/KDH";
+import NES from "../components/nes/NES";
+import KYE from "../components/kye/KYE";
+import KYH from "../components/kyh/KYH";
+import PGB from "../components/pgb/PGB";
+
+function MainTest() {
+  return (
+    <div>
+      <Link to={`/kdh`} element={<KDH />}>
```

Comment:
저는 일반 문자열에 백틱을 쓰는건 좋지 않다고 생각해요~
```suggestion
      <Link to='/kdh' element={<KDH />}>
```